src/ORM/Sortable/SortableSubscriber.php

```diff
@@ -43,35 +33,21 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($this->isSortable($classMetadata)) {
-
+        if (is_subclass_of($classMetadata->getName(), 'Knp\DoctrineBehaviors\Model\Sortable\SortableInterface')) {
```

Comment:
Wouldn't you prefer basing yourself on the *Properties trait to trigger the mapping ? Using the interface might cause problems in case of inheritance (CF #317).

What do you think about the pattern I used in my PRs (e.g. https://github.com/KnpLabs/DoctrineBehaviors/pull/322/files#diff-9af172be101c124fe7673df513efc25aL296)